Wave/Wave/Section.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  Section.swift
+//  Wave
+//
+//  Created by Jimmy McDermott on 6/27/16.
+//  Copyright © 2016 Blue Line Labs. All rights reserved.
+//
+
+import Foundation
+
+public class Section: SectionType {
+    public var name: String = ""
+    var settings = [Setting<Any>]()
+    
+    public init() { }
+    
+    public func registerCells(tableView: UITableView) {
+        for setting in settings {
+            setting.registerCells(tableView)
+        }
+    }
+    
+    public func numberOfRows() -> Int {
+        return settings.count
+    }
+    
+    public func tableViewCellIdentifier(indexPath: NSIndexPath) -> String {
+        return settings[indexPath.row].cellIdentifier
+    }
+    
+    public class func addRow() -> Section {
+        return Section()
```

Comment:
Why is a new section being returned here? I was thinking we'd just return self so it ends up being more of a builder pattern.